.github/depends.R

```diff
@@ -0,0 +1,28 @@
+
+# these dependencies casuing tidyverse not to work?
+install.packages(
```

Comment:
GHA on the ubuntu runner was failing to install tidyverse because it could not install those packages. Not sure why, but installing them first separately fixed it